ffi/java-idsdk/src/main/java/com/iotics/sdk/identity/jna/JnaSdkApiInitialiser.java

```diff
@@ -0,0 +1,27 @@
+package com.iotics.sdk.identity.jna;
+
+import com.sun.jna.Native;
+
+import java.io.File;
+
+public class JnaSdkApiInitialiser {
+    private static String LIB_PATH = new File("./bin/lib-iotics-id-sdk.so").getAbsolutePath();
```

Comment:
Will this be configurable?.. or is this just a for-Linux exampe?

Reply:
yes - use the second constructor